app/models/BannerTests.scala

```diff
@@ -75,6 +75,7 @@ case class BannerTest(
     minArticlesBeforeShowingBanner: Int,
     userCohort: UserCohort,
     locations: List[Region] = Nil,
+    contextTargeting: Option[ContextTargeting] = Some(ContextTargeting(Nil,Nil,Nil,Nil)),
```

Comment:
existing banner tests will not have this field, so set a default with no targeting